packages/react-components/src/components/CheckBoxGroup.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useRuleEngine, useFormIntl } from '@aemforms/af-react-renderer';
+import {State, FieldJson} from '@aemforms/af-core';
+
+const CheckBoxGroup = (fieldjson: State<FieldJson>) => {
+   const [props, handlers] = useRuleEngine(fieldjson);
+   const { id, label, required, enumNames, enum: enums, value, name, valid, errorMessage, description } = props;
+   const i18n = useFormIntl();
+   const newVal = !value ? [] : value instanceof Array ? value : [value];
+   const options = enumNames && enumNames.length ? enumNames : enums || [];
+   const validateState = valid === false ? 'invalid' : 'valid';
+   const error = validateState === 'invalid';
+   const formattedDescription = i18n.formatMessage({ id: `${description}` });
```

Comment:
This will not work. Have to test it?
You have to get the localization id from "props.properties['afs:translationIds']"

Reply:
Implemented!